cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/CgmesPhaseTapChangerBuilder.java

```diff
@@ -142,8 +142,12 @@ private void addStepsAsymmetrical() {
             return;
         }
         double alphaMax = tapChanger.getSteps().stream().map(TapChanger.Step::getAngle)
-                .mapToDouble(Double::doubleValue).max().orElse(Double.NaN);
+                .mapToDouble(Double::doubleValue).max().orElse(0);
         tapChanger.getSteps().forEach(step -> {
+            if (alphaMax == 0 || xtx == 0) {
+                step.setX(0);
+                return;
+            }
```

Comment:
I think the problem is that the `X `of the transformer is zero. We should not allow transformers with zero reactance. I think is better to fix this problem in the class `CgmesT2xModel `(Line 35) for `twoWindingsTransformer `and in the class `CgmesT3xModel `(Line 43) for `threeWindingsTransformers`. We can write an invalid warning and fix the `X `to a residual value (0.0001 by example) or maybe we can throw an exception.

Reply:
Is it okay for you to put `Double.MIN_VALUE`? I would rather correct the value than throw an exception (the validation can be modified in another PR)

Reply:
For clarification, I ran into this exception with `alphaMax` equals to 0.

Reply:
Sorry, then my comment does not make sense. Maybe is better to check only alphaMax
```
if (alphaMax == 0) {
                step.setX(0);
                return;
            }
```

Reply:
I did what you suggested